adapter/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "polkadot-adapter",
+  "version": "1.0.0",
+  "main": "index.ts",
+  "author": "NodeFactory",
+  "license": "MIT",
+  "dependencies": {
+    "@polkadot/extension-inject": "^0.23.1"
+  }
```

Comment:
You should copy build scripts and dependencies from something like nodefatoryio/js-libp2p-noise